### PR TITLE
Allow PR monitor to run for multiple teams.

### PR DESCRIPTION
**Pull request by MancunianSam**
There are environments on GitHub for tdr and DRI and a new environment variable in the application.conf file.

### CODE REVIEW

File: .github/workflows/run.yml [hunk 1]
```diff
@@ -4,12 +4,27 @@ on:
   schedule:
     - cron: '2 7 * * *'
 jobs:
-  run:
+
```
Comment (TomJKing) [inline .github/workflows/run.yml:7]: Unnecessary line break

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment. Thanks

File: .github/workflows/run.yml (hunk 1 above)
Comment (MancunianSam) [reply .github/workflows/run.yml:7]: Three days away and I forget the line breaks :smile: I've removed it.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**